pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/AbstractLombokAwareRule.java

```diff
@@ -30,14 +32,14 @@
     private static final Set<String> LOMBOK_ANNOTATIONS = new HashSet<>();
 
     static {
-        LOMBOK_ANNOTATIONS.add("Data");
-        LOMBOK_ANNOTATIONS.add("Getter");
-        LOMBOK_ANNOTATIONS.add("Setter");
-        LOMBOK_ANNOTATIONS.add("Value");
-        LOMBOK_ANNOTATIONS.add("RequiredArgsConstructor");
-        LOMBOK_ANNOTATIONS.add("AllArgsConstructor");
-        LOMBOK_ANNOTATIONS.add("NoArgsConstructor");
-        LOMBOK_ANNOTATIONS.add("Builder");
+        LOMBOK_ANNOTATIONS.add("lombok.Data");
+        LOMBOK_ANNOTATIONS.add("lombok.Getter");
+        LOMBOK_ANNOTATIONS.add("lombok.Setter");
+        LOMBOK_ANNOTATIONS.add("lombok.Value");
+        LOMBOK_ANNOTATIONS.add("lombok.RequiredArgsConstructor");
+        LOMBOK_ANNOTATIONS.add("lombok.AllArgsConstructor");
+        LOMBOK_ANNOTATIONS.add("lombok.NoArgsConstructor");
+        LOMBOK_ANNOTATIONS.add("lombok.Builder");
```

Comment:
I'd try to build this on top of the new property.

Reply:
Done.